joomla/joomla-cms
# Adds jlayout searchtools to com_users user groups

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the search tools to com_users user groups.

As an extra also added a badge and a link in the number of users in that group.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/12934064/cf36912e-cf83-11e5-90ac-0e3efc0aafce.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12934050/b0679fae-cf83-11e5-93bc-0d040c902779.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Users -> User Groups
3. Check the ordering select box and search is working fine.
